spdx/writers/jsonyamlxml.py

```diff
@@ -116,16 +116,20 @@ def create_package_info(self, package):
         package_object["SPDXID"] = self.spdx_id(package.spdx_id)
         package_object["name"] = package.name
         package_object["downloadLocation"] = package.download_location.__str__()
-        package_object["packageVerificationCode"] = self.package_verification_code(
-            package
-        )
+        if package.files_analyzed in [True, None]:
```

Comment:
as the default value for this field is True, we should initialize the field to True.

this test is really not obvious, and will lead to bugs in the future